group_vars/maintenance.yml

```diff
@@ -11,9 +11,9 @@ galaxy_log_dir: "/var/log/galaxy"
 
 galaxy_config:
   galaxy:
-    job_working_directory: /data/jwd04/main
-    nginx_upload_store: "/data/jwd04/nginx_upload/main/uploads"
-    nginx_upload_job_files_store: "/data/jwd04/nginx_upload/main/jobfiles"
+    job_working_directory: "{{ job_working_root_dir }}/main/"
```

Comment:
Has the maintenance.yml file access to the all.yml?

Reply:
Yes, `all.yml` is included in the `maintenance` playbook.